src/map/core.rs

```diff
@@ -80,6 +80,7 @@ fn update_index(table: &mut Indices, hash: HashValue, old: usize, new: usize) {
 /// and without regard for duplication.
 ///
 /// ***Panics*** if there is not sufficient capacity already.
+#[track_caller]
 fn insert_bulk_no_grow<K, V>(indices: &mut Indices, entries: &[Bucket<K, V>]) {
```

Comment:
This is an internal function, and the pre-condition is entirely in our own control. If this ever fails, I think it would be confusing to the user to have their caller location reported, when they really should report a bug to us. So, I don't think we should add this one.